db/migrate/20151208150956_fix_host_storage_replication_on_upgrade.rb

```diff
@@ -9,36 +8,10 @@ class HostsStorage < ActiveRecord::Base
   end
 
   def up
-    if on_replication_target?
-      run_for_replication_target
-    else
-      run_for_replication_source
-    end
+    HostsStorage.delete_all unless on_replication_target?
```

Comment:
I think this is wrong...  The previous code did if `run_for_replication_target if on_replication_target?`, but this changed it to unless.